remediation_worker/jobs/aws_s3_remove_fullaccess_authenticatedusers/README.md

```diff
@@ -0,0 +1,68 @@
+# Remove S3 Public Admin ACL
+
+This job will remove the public "full access privileges" permission. All other ACL permissions will be left alone. This means that if "AuthenticatedUsers" have access to FULL_CONTROL, the permissions will be changed to remove the 'FULL_CONTROL' permission. 
+
+### Applicable Rule
+
+##### Rule ID:
+5c8c26617a550e1fb6560c69
```

Comment:
Rule Id for s3 full access for authenticated users is this - 5c8c26567a550e1fb6560c5d